website/docs/core-concepts/connecting-to-data-sources/authentication/authentication-type/basic-authentication.md

```diff
@@ -1,7 +1,41 @@
 # Basic
 
-To share basic authentication details across multiple APIs, you can use the `Basic` authentication mechanism within datasources. To do this, click on the `+` icon next to the datasources section and add an Authenticated API. Enter the Base URL of the API (ex. **https://airtable.com**) and select the authentication type as **Basic**.
+Basic authentication is a simple method for verifying the identity of users making requests to an API. It involves sending an HTTP header with a base64-encoded username and password. This process is simple to set up and configure, making it convenient for developers to implement in their APIs.
 
-The username and password can now be configured under the authentication section and will add the base64 version of the username:password to the header of all APIs added under this datasource. Save this datasource and add a new API
+## How does basic authentication work
 
-![Select Basic from the available Authentication Type](</img/auth_api.gif>)
+The `base64(username:password)` portion is the base64-encoded version of the string username:password. This allows the server to verify the identity of the user making the request without sending the `plaintext` password over the network.
+
+When the server receives a request with the Authorization header, it decodes the base64-encoded string and checks the `username` and `password` against its own records. If the credentials are valid, the server allows the request to proceed. If the credentials are invalid or missing, the server returns an **HTTP 401** Unauthorized error to the client.
+
+To share basic authentication details across multiple APIs, you can use the `Basic` authentication mechanism within datasources. To do this:
+
+ <VideoEmbed host="youtube" videoId="SM0K4XkkJgM" title="basic authentication" caption="Basic Authentication"/> 
+
+* Click on the `+` icon next to the datasources section 
+* Add an Authenticated API
+* Enter the Base URL of the API (example: `https://mock-api.appsmith.com`) and select the authentication type as **Basic**.
+
+The base64-encoded username and password are automatically added to the header of the API request, allowing the API to verify the identity of the user making the request.
+
+
+
+
+
+## Authentication
+
+
+![Select Basic from the available Authentication Type](</img/basic-auth.png>)
+
+
+### Username
+In basic authentication, the username is a string that identifies the user making the request to the API. It's usually a combination of the user's first and last **name**, or it could be a unique identifier such as an **email address** or **employee number**.
```

Comment:
As the page talks about Basic Authentication, we need not specify it again for each param:
```suggestion
The username is a string that identifies the user making the request to the API. It is often a combination of the user's first and last name, or it could be a unique identifier like an email address or employee number.
```